packages/compass-indexes/src/components/indexes-table/indexes-table.tsx

```diff
@@ -134,17 +135,17 @@ export const IndexesTable: React.FunctionComponent<IndexesTableProps> = ({
               properties={index.properties}
             />
           </Cell>
-          {/* Delete column is conditional */}
-          {index.name !== '_id_' && canDeleteIndex && (
-            <Cell data-testid="index-drop-field" className={cellStyles}>
-              <div className={cx(deleteFieldStyles, 'delete-cell')}>
-                <DropField
-                  name={index.name}
-                  onDelete={() => onDeleteIndex(index.name)}
-                />
+          <Cell data-testid="index-actions-field" className={cellStyles}>
```

Comment:
@mabaasit FYI this fixes the missing cell for `_id` too

Reply:
Linking ticket here to close once this is approved: https://jira.mongodb.org/browse/COMPASS-6083